python/lsst/daf/butler/formatters/fitsExposureFormatter.py

```diff
@@ -96,8 +76,8 @@ def stripMetadata(self):
         bboxFromMetadata(self.metadata)  # always strips
         makeSkyWcs(self.metadata, strip=True)
 
-    def readInfoComponent(self, component):
-        """Read a component held by ExposureInfo.
+    def readComponent(self, component):
```

Comment:
I think this is going to have to take a parameters argument so that you can do things like pass a bounding box to readMask.

Reply:
Added here, and to read(), with appropriate handling to ensure parameters are not passed to methods that do not expect parameters.